Core/Inc/ws2812.hpp

```diff
@@ -2,14 +2,24 @@
  * ws2812.h
  *
  *  Created on: Sep 22, 2024
- *      Author: anni and Fola
- *      Based on
+ *      Author: anni
```

Comment:
You forgot to add yourself!!!!
```suggestion
 *      Author: Anni, Fola
```